index.js

```diff
@@ -1,24 +1,79 @@
-function rot13(str) { // LBH QVQ VG!
-var shifted = 13;
-var splitted = str.split('');
-var arr = [];
-var inCharCode = splitted.forEach(function(element) {
```

Comment:
There is a map function. Use it if you want to convert each array element to something else.